src/world/physicsReferences.lua

```diff
@@ -76,7 +76,10 @@ function PhysicsReferences.getCategory(type)
 end
 
 function PhysicsReferences.setFixtureType(fixture, type)
-	fixture:setSensor(sensors[type])
+	fixture:setSensor(isSensor[type])
+	if isSensor[type] then
```

Comment:
I think we need to better define what this file is for. Before this change, it was basically only collision filters. With this, it becomes...a bunch of stuff related to physics objects? I know this is a convenient place to put this code, but I think we need to step back and think about how to design this a little more. Maybe PhysicsReferences will need to be broken up in to a couple single-purpose classes.